docs/solutions/Hadoop/EL7/Cloudera/Development/variables.tf

```diff
@@ -0,0 +1,38 @@
+###
+## Variables here are sourced from env, but still need to be initialized for Terraform
+###
+
+variable "tenancy_ocid" {}
+variable "user_ocid" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+variable "region" { default = "us-phoenix-1" }
+
+variable "compartment_ocid" {}
+variable "ssh_public_key" {}
+variable "ssh_private_key" {}
+
+variable "AD" { default = "2" }
+
+variable "image_ocid" {
+  default = " "
```

Comment:
Is this a valid value? If not, then don't set it as a default.

Reply:
Noted - fixing.